src/js/styles/generateStyles.js

```diff
@@ -43,7 +45,7 @@ export function generateStyles(options) {
       '&:hover': {
         background: darken(0.1, variables.shepherdThemePrimary)
       },
-      '&.shepherd-button-secondary': {
+      [`&.${classPrefix}shepherd-button-secondary`]: {
```

Comment:
This class is not generated by shepherd, but passed in as a custom class, so I don't think it should get prefixes, unless we want to support secondary buttons, which maybe we should?

Reply:
But the style is still part of the library, so I think it should be prefixes like all the others.

I do believe it would be more intuitive, instead of having a "hidden" class for secondary button, to have a more expressive way of setting a button as secondary via configuration.

Reply:
I definitely agree. We should expose an option to make a button secondary.